src/Symfony/Bundle/FrameworkBundle/Command/WorkflowDumpCommand.php

```diff
@@ -12,120 +12,11 @@
 namespace Symfony\Bundle\FrameworkBundle\Command;
 
 use Symfony\Component\Console\Attribute\AsCommand;
-use Symfony\Component\Console\Command\Command;
-use Symfony\Component\Console\Completion\CompletionInput;
-use Symfony\Component\Console\Completion\CompletionSuggestions;
-use Symfony\Component\Console\Exception\InvalidArgumentException;
-use Symfony\Component\Console\Input\InputArgument;
-use Symfony\Component\Console\Input\InputInterface;
-use Symfony\Component\Console\Input\InputOption;
-use Symfony\Component\Console\Output\OutputInterface;
-use Symfony\Component\DependencyInjection\ServiceLocator;
-use Symfony\Component\Workflow\Debug\TraceableWorkflow;
-use Symfony\Component\Workflow\Dumper\GraphvizDumper;
-use Symfony\Component\Workflow\Dumper\MermaidDumper;
-use Symfony\Component\Workflow\Dumper\PlantUmlDumper;
-use Symfony\Component\Workflow\Dumper\StateMachineGraphvizDumper;
-use Symfony\Component\Workflow\Marking;
-use Symfony\Component\Workflow\StateMachine;
+use Symfony\Component\Workflow\Command\WorkflowDumpCommand as BaseWorkflowDumpCommand;
+
+trigger_deprecation('symfony/framework-bundle', '7.4', 'The "%s" class is deprecated, use "%s" instead.', WorkflowDumpCommand::class, BaseWorkflowDumpCommand::class);
 
-/**
```

Comment:
instead of removing the phpdoc entirely, you should add the `@deprecated` phpdoc.